### ISSUE TITLE
(Feature request) Key-binding for show/hide to tray

### ISSUE THREAD

**Issue by szmaszmo1** [TriliumNext/Trilium#3043]
### Describe feature

Would it be possible to provide a new key-binding option e.g. by "Keyboard shortcuts" for show/hide to tray?

In this way, it would be quicker (e.g. by pressing "alt+z" on keyboard) to bring application to front and hide it back to tray, as there would be no need to click the tray icon and click on menu anymore. Somewhat sometimes just clicking icon is enough (on KDE Plasma 5.24.x), but even this is slower than keyboard.

I remember this feature from keynote/keynote-nf (https://github.com/dpradov/keynote-nf) project. 
This project is much much better, but I just miss this "small" functionality..

Thanks a lot for this great job, greetings!

**Comment by banjuer** [comment]
in version 0.63.5, toggleTray shortcut not works well, it can hide but can not be display

**Comment by zadam** [comment]
That's right, now thinking about it, it cannot work like this to restore the window. It would have to register the shortcut "globally" to be able to reopen the windows.

**Comment by themql** [comment]
https://github.com/TriliumNext/Notes/issues/516#issuecomment-2436745673

**Comment by eliandoran** [comment]
Done in https://github.com/TriliumNext/Trilium/pull/4267.
